### PR TITLE
Fix CodeQL alerts: redundant check

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rintaro.okamura@gmail.com>

### Description:

CodeQL detects these alerts.
- Redundant check for negative value
This PR fixes them.

### Related Issue:

Nothing

### How Has This Been Tested?:

Nothing

### Environment:

- Go Version: 1.16
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [X] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [X] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pkg/agent/core/ngt/handler/grpc/handler.go [hunk 1]
```diff
@@ -1047,7 +1047,7 @@ func (s *server) MultiUpsert(ctx context.Context, reqs *payload.Upsert_MultiRequ
 }
 
 func (s *server) Remove(ctx context.Context, req *payload.Remove_Request) (res *payload.Object_Location, err error) {
-	ctx, span := trace.StartSpan(ctx, apiName+".Remove")
+	_, span := trace.StartSpan(ctx, apiName+".Remove")
```
Comment (kpango) [inline pkg/agent/core/ngt/handler/grpc/handler.go:1050]: StartSpan often generates a new context by context.WithValue with embedded traceID and other values in context. I think it is better to ignore this part even CodeQL warns it.

**Review by kpango — CHANGES_REQUESTED** [review]
only internal/net/net.go seems okay, can you revert others??

**Comment by rinx** [comment]
@kpango I've updated it!

**Review by kpango — APPROVED** [review]
LGTM

**Pull request merged**